web/frontend/src/lib/camera-state.ts

```diff
@@ -1,2 +1,15 @@
-export const baseStreamStates = new Map<string, boolean>();
-export const cameraStreamStates = new Map<string, boolean>();
+interface StreamState {
+  'on' : boolean,
+  'live' : boolean,
+  'name' : string
+}
+
+export const cameraStreamStates = new Map<string, StreamState>();
+
+export const selectedMap = {
```

Comment:
One last nit, let's rename this `cameraRefreshMap` or something to be a bit more explicit what this is.